catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/builder/AttributeValueNormalizer.java

```diff
@@ -76,9 +76,36 @@ public String normalizeForJson(String property, String value) {
     if (instant != null) {
       return instant.toString();
     }
+
+    String isoDateRange = getIsoDateRangeFromEpoch(value);
+    if (isoDateRange != null) {
+      return isoDateRange;
+    }
+
     return value;
   }
 
+  private String getIsoDateRangeFromEpoch(String value) {
```

Comment:
Since this can return null, I think it's a good idea annotate the return type with `javax.annotation.Nullable`